config-templates/netapp/netapp-ontap-san/21.04/README.md

```diff
@@ -0,0 +1,113 @@
+# NetApp Trident - ONTAP SAN Driver
+
+You can use the `netapp-ontap-san` Satellite storage template to deploy NetApp storage drivers which you can use dynamically provision and manage iSCSI LUNs in your ONTAP storage.
+
+## Prerequisites
+
+**Planning considerations for the Infrastructure Admin**
+* Create a cluster that meets the requirements for ONTAP SAN. For more information, see the [NetApp documentation](https://netapp-trident.readthedocs.io/en/stable-v21.04/support/requirements.html). Verify that your backend ONTAP cluster is configured as a Trident backend.
+   * You must have a dedicated Storage Virtual Machine (SVM) for Trident. Volumes and LUNs that are created by Trident are created in this SVM.
+   * You must have one or more aggregates assigned to the SVM.
+   * You must have one or more dataLIFs for the SVM.
+   * You must have iSCSI services enabled on the SVM.
+   * You must set up a snapshot policy on the SVM.
+* Share following details with Location Admin.
+
+
+**Planning consideration for Location Admin**
+* Make sure `netapp-trident` is deployed on the target cluster.
+* Review the template parameters and retrieve the values from your NetApp cluster.
+
+## NetApp Ontap-SAN Driver parameters & how to retrieve them
+
+List the template parameters.
+```
+ibmcloud sat storage template get --name netapp-ontap-san --version 21.04
+```
+
+**NetApp Ontap-SAN Driver parameters**
+
+| Parameter name | Required? | Description | Default value if not provided |
+| --- | --- | --- | --- |
+| `managementLIF` | Required | The IP address of the management LIF. Example: `10.0.0.1`. | N/A |
+| `dataLIF` | Required | The IP address of data LIF. Example: `10.0.0.2`. | N/A | 
+| `svm` | Required | The name of the storage virtual machine. Example: `svm-iscsi`. | N/A | 
+| `username` | Required | The username to connect to the storage device. | N/A |
+| `password` | Required | The password to connect to the storage device. | N/A |
+| `limitVolumeSize` | Optional | Maximum volume size that can be requested and qtree parent volume size. Example: `50Gi`| `(not enforced by default)` |
+| `limitAggregateUsage` | Optional | Limit provisioning of volumes if parent volume usage exceeds this value. For example, if a volume is requested that causes parent volume usage to exceed this value, the volume provisioning fails. Example: `80%`  | `(not enforced by default)` |
+
+
+## Default storage classes
+
+The following storage classes are installed when you assign your `netapp-ontap-san` configuration to your clusters to allow you to take advantage of ONTAP's QoS features:
+
+| Storage class name | Type | File system | IOPs | Reclaim policy |
+| --- | --- | --- | --- | --- |
+| `ntap-block-gold` | Ontap-SAN | Block | user defined**\*** | Delete |
+| `ntap-block-silver` | Ontap-SAN | Block | user defined**\*** | Delete |
+| `ntap-block-bronze` | Ontap-SAN | Block | user defined**\*** | Delete | 
+| `ntap-block-default` | Ontap-SAN | Block | n/a | Delete | 
```

Comment:
Update the storage class names to `sat-netapp-block-gold`,`sat-netapp-block-silver`, `sat-netapp-block-bronze` and `sat-netapp-block-default`